.github/workflows/pull-request-automation.yml

```diff
@@ -29,11 +29,13 @@ jobs:
               with:
                   # npm cache files are stored in `~/.npm` on Linux/macOS
                   path: ~/.npm
-                  key: ${{ runner.os }}-node-${{ matrix.node }}-npm-cache-${{ hashFiles('**/package-lock.json') }}
+                  key: ${{ runner.os }}-node-${{ matrix.node }}-npm-pr-automation-cache-${{ hashFiles('**/package-lock.json') }}
```

Comment:
Do you think jobs that use `npm ci` and `npm install` should have different names as well?

Reply:
I think it should be OK to share caches between those workflows.

The cache is not the `node_modules` folder, but rather the global level cache on the machine in `~/.npm`. The only difference between the two is that `npm ci` it installs the exact packages listed in the lock file, and will return an error if a needed dependency is not found in that lock file. If the dependency is not present in the cache when using either command, it will just fetch it from NPM.

We also guard against discrepancies between the two by running `npm install` in the [static analysis workflow](https://github.com/WordPress/gutenberg/blob/trunk/.github/workflows/static-checks.yml#L39) and checking for local changes.